src/Bridges/NetteDI/MigrationsExtension.php

```diff
@@ -28,6 +28,8 @@ class MigrationsExtension extends Nette\DI\CompilerExtension
 	/** @var array */
 	protected $dbals = [
 		'dibi' => 'Nextras\Migrations\Bridges\Dibi\DibiAdapter',
+		'dibi2' => 'Nextras\Migrations\Bridges\Dibi\Dibi2Adapter',
+		'dibi3' => 'Nextras\Migrations\Bridges\Dibi\Dibi3Adapter',
```

Comment:
I'm currently unsure if we should treat `Dibi2Adapter` and `Dibi3Adapter` as internal classes or whether we should expose them like this.